recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
```

Comment:
Since, right now, only `tensorflow` use `bazel` I would bother too much with this. If we start making heavy use of it though we should re-visit this.